content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -114,13 +114,13 @@ the name of the MySQL Service defined above, and WordPress will access the datab
       
 3. Add them to `kustomization.yaml` file.
 
-      ```shell
-      cat <<EOF >>./kustomization.yaml
-      resources:
-        - mysql-deployment.yaml
-        - wordpress-deployment.yaml
-      EOF
-      ```
+```shell
+cat <<EOF >>./kustomization.yaml
```

Comment:
One possibility is to show the code as a yaml example and remove the `cat` command.
For example:
```
     ```yaml
      resources:
        - mysql-deployment.yaml
        - wordpress-deployment.yaml
      ```
```

Reply:
This seems a valid option too, as it can avoid the confusion caused by the indentation.
But a down side to it, is losing the fluid usage follow up of the documentation as the user will have to figure out how to append the file using a terminal (but I guess who ever it learning K8s should have a least some basic terminal knowledge)

Reply:
You can use a `codenew` shortcode. See https://github.com/kubernetes/website/pull/18245 for a PR that's aimed at documenting good practice for the style guide.

And then, eg, suggest that the reader runs: `curl -L https://k8s.io/example/foo.yaml >> kustomization.yaml`

This teaches appending without also teaching heredocs.